apps/web/pages/video/meeting-ended/[uid].tsx

```diff
@@ -4,21 +4,19 @@ import dayjs from "dayjs";
 import { NextPageContext } from "next";
 import { getSession } from "next-auth/react";
 import { useRouter } from "next/router";
-import { useState } from "react";
 import { useEffect } from "react";
 
 import prisma from "@lib/prisma";
+import { detectBrowserTimeFormat } from "@lib/timeFormat";
 import { inferSSRProps } from "@lib/types/inferSSRProps";
 
 import { HeadSeo } from "@components/seo/head-seo";
 import Button from "@components/ui/Button";
 
 export default function MeetingUnavailable(props: inferSSRProps<typeof getServerSideProps>) {
   const router = useRouter();
-
-  const [is24h, setIs24h] = useState(false);
-
-  //if no booking redirectis to the 404 page
+  
```

Comment:
This line is causing a linting error

Reply:
Gracias @zomars ahora lo arreglo y pusheo!